apps/ssr-tests-v9/package.json

```diff
@@ -13,7 +13,7 @@
     "clean": "just-scripts clean",
     "code-style": "just-scripts code-style",
     "lint": "just-scripts lint",
-    "test": "jest --passWithNoTests",
+    "test": "jest && node -r @fluentui/scripts/ts-node-register ./src/test.ts",
```

Comment:
hmm IMO this clouds the actual intent of the commands for the app. From DX point of view I expect to run the `ssr` test not tests for actual implementation. we can go various ways with this. 

lets keep it as is for now, as a followup it would be great if we can extract all the logic to a separate package/cli tool and then use that tool for this app.

Reply:
Yeah, don't really like that, but it's better to have tests than don't 😀